source/_components/tradfri.markdown

```diff
@@ -14,44 +14,42 @@ ha_iot_class: "Local Polling"
 ha_release: 0.43
 ---
 
-The `tradfri` component supports for the IKEA Trådfri (Tradfri) gateway. The gateway can control lights connected to it and Home Assistant will automatically discover its presence on your network. 
+The `tradfri` component supports for the IKEA Trådfri (Tradfri) gateway. The gateway can control lights connected to it and Home Assistant will automatically discover its presence on your network.
 
-For this to work, you need to install a modified lib-coap library.
+For this to work, you need to install `aiocoap` and `tinydtls`.
 
-<p class='note warning'>
-This component does **not** work on Windows, as the modified lib-coap doesn't exists for Windows.
-</p>
 <p class='note'>
 If you are using [Hass.io](/hassio/) then just move forward to the configuration as all requirements are already fulfilled.
 </p>
 
-Linux:
+Installation script:
 
 ```bash
-$ sudo apt-get install libtool
-$ sudo apt-get install autoconf
-
-$ git clone --depth 1 --recursive -b dtls https://github.com/home-assistant/libcoap.git
-$ cd libcoap
-$ ./autogen.sh
-$ ./configure --disable-documentation --disable-shared --without-debug CFLAGS="-D COAP_DEBUG_FD=stderr"
-$ make
-$ sudo make install
-```
+#!/bin/sh
+# Installs a modified coap client with support for dtls for use with IKEA Tradfri
 
-macOS:
+# Stop on errors
+set -e
 
-```bash
-$ brew install libtool
-$ brew install autoconf
-$ brew install automake
-$ git clone --depth 1 --recursive -b dtls https://github.com/home-assistant/libcoap.git
-$ cd libcoap
-$ ./autogen.sh
-$ ./configure --disable-documentation --disable-shared --without-debug CFLAGS="-D COAP_DEBUG_FD=stderr"
-$ make
-$ make install
+python3 -m pip install cython
+
+cd /usr/src/app/
```

Comment:
Not everybody has `/usr/src/app`. This script will fail for them. I'd either `mkdir` it or use `tmp`. The source is not needed after the installation, anyway.

Reply:
Good spot.